Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -25,6 +24,144 @@
 #include "VideoCommon/VertexShaderGen.h"
 #include "VideoCommon/VideoBackendBase.h"
 
+constexpr const char* GLSL_REINTERPRET_PIXELFMT_SINGLELAYER = R"GLSL(
```

Comment:
These two could be collapsed into one shader with a macro, e.g. #if MULTILAYER. But that's a larger change, so I'm fine to keep these separate for now.